docs/ops/sequence/AUGRUSequence_dev.md

```diff
@@ -0,0 +1,168 @@
+  * **Type**: a list of strings
+  * **Default value**: *sigmoid* for f, *tanh* for g
+  * **Required**: *no*
+  * **Supported value**: *sigmoid* for f, *tanh* for g
+
+* *activations_alpha, activations_beta*
+
+  * **Description**: *activations_alpha, activations_beta* attributes of functions; applicability and meaning of these attributes depends on chosen activation functions
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: []
+  * **Required**: *no*
+  * **Supported value**: []
+
+* *clip*
+
+  * **Description**: *clip* specifies bound values *[-C, C]* for tensor clipping. Clipping is performed before activations.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: `0.` that means the clipping is not applied
+  * **Required**: *no*
+  * **Supported value**: `0.`
+
+* *direction*
+
+  * **Description**: Specify if the RNN is forward, reverse, or bidirectional. If it is one of *forward* or *reverse* then `num_directions = 1`, if it is *bidirectional*, then `num_directions = 2`. This `num_directions` value specifies input/output shape requirements.
+  * **Range of values**: *forward*, *reverse*, *bidirectional*
+  * **Type**: `string`
+  * **Default value**: *forward*
+  * **Required**: *no*
+  * **Supported value**: *forward*
+
+* *linear_before_reset*
+
+  * **Description**: *linear_before_reset* flag denotes, if the output of hidden gate is multiplied by the reset gate before or after linear transformation.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+  * **Supported value**: False
+
+**Inputs**
+
+* **1**: `X` - 3D tensor of type *T1* `[batch_size, seq_length, input_size]`, input data. **Required.**
+
+* **2**: `H_t` - 3D tensor of type *T1* and shape `[batch_size, num_directions, hidden_size]`. Input with initial hidden state data. **Required.**
+
+* **3**: `sequence_lengths` - 1D tensor of type *T2* and shape `[batch_size]`. Specifies real sequence lengths for each batch element. **Required.**
+
+* **4**: `W` - 3D tensor of type *T1* and shape `[num_directions, 3 * hidden_size, input_size]`. The weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **5**: `R` - 3D tensor of type *T1* and shape `[num_directions, 3 * hidden_size, hidden_size]`. The recurrence weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **6**: `B` - 2D tensor of type *T*. The biases. If *linear_before_reset* is set to  `False`, then the shape is `[num_directions, 3 * hidden_size]`, gate order: zrh. Otherwise the shape is `[num_directions, 4 * hidden_size]` - the sum of biases for z and r gates (weights and recurrence weights), the biases for h gate are placed separately. **Required.**
+
+* **7**: `A` - 3D tensor of type *T* `[batch_size, seq_length, 1]`, the attention score. **Required.**
```

Comment:
we don't have type *T* in the Types section below :)
the same for B input

Reply:
Sure, it should be *T1* as well